org.eclipse.lemminx/src/main/resources/org/eclipse/lemminx/services/snippets/processing-instruction-snippets.json

```diff
@@ -0,0 +1,24 @@
+{
+	"Insert XML Model: XSD": {
+	  "prefix": [
+		"<?xml-model"
+	  ],
+	  "suffix": "?>",
+	  "body": [
+		"<?xml-model href=\"${1:file.xsd}\" type=\"application/xml\" schematypens=\"http://www.w3.org/2001/XMLSchema\"?>${0}"
+	  ],
+	  "label": "$description",
```

Comment:
I think we should have a shorter label. Here a suggestion

`"label": "Insert XML Schema association",`